hw/xbox/nv2a/nv2a_psh.c

```diff
@@ -49,120 +49,128 @@
-    PS_REGISTER_FOG=               0x03L, // r
-    PS_REGISTER_V0=                0x04L, // r/w
-    PS_REGISTER_V1=                0x05L, // r/w
-    PS_REGISTER_T0=                0x08L, // r/w
-    PS_REGISTER_T1=                0x09L, // r/w
-    PS_REGISTER_T2=                0x0aL, // r/w
-    PS_REGISTER_T3=                0x0bL, // r/w
-    PS_REGISTER_R0=                0x0cL, // r/w
-    PS_REGISTER_R1=                0x0dL, // r/w
-    PS_REGISTER_V1R0_SUM=          0x0eL, // r
-    PS_REGISTER_EF_PROD=           0x0fL, // r
-
-    PS_REGISTER_ONE=               PS_REGISTER_ZERO | PS_INPUTMAPPING_UNSIGNED_INVERT, // OK for final combiner
-    PS_REGISTER_NEGATIVE_ONE=      PS_REGISTER_ZERO | PS_INPUTMAPPING_EXPAND_NORMAL,   // invalid for final combiner
-    PS_REGISTER_ONE_HALF=          PS_REGISTER_ZERO | PS_INPUTMAPPING_HALFBIAS_NEGATE, // invalid for final combiner
-    PS_REGISTER_NEGATIVE_ONE_HALF= PS_REGISTER_ZERO | PS_INPUTMAPPING_HALFBIAS_NORMAL, // invalid for final combiner
+enum PS_REGISTER {
+    PS_REGISTER_ZERO = 0x00L, // r
+    PS_REGISTER_DISCARD = 0x00L, // w
+    PS_REGISTER_C0 = 0x01L, // r
+    PS_REGISTER_C1 = 0x02L, // r
+    PS_REGISTER_FOG = 0x03L, // r
+    PS_REGISTER_V0 = 0x04L, // r/w
+    PS_REGISTER_V1 = 0x05L, // r/w
+    PS_REGISTER_T0 = 0x08L, // r/w
+    PS_REGISTER_T1 = 0x09L, // r/w
+    PS_REGISTER_T2 = 0x0aL, // r/w
+    PS_REGISTER_T3 = 0x0bL, // r/w
+    PS_REGISTER_R0 = 0x0cL, // r/w
+    PS_REGISTER_R1 = 0x0dL, // r/w
+    PS_REGISTER_V1R0_SUM = 0x0eL, // r
+    PS_REGISTER_EF_PROD = 0x0fL, // r
+
+    PS_REGISTER_ONE = PS_REGISTER_ZERO |
+                      PS_INPUTMAPPING_UNSIGNED_INVERT, // OK for final combiner
+    PS_REGISTER_NEGATIVE_ONE =
+        PS_REGISTER_ZERO |
+        PS_INPUTMAPPING_EXPAND_NORMAL, // invalid for final combiner
+    PS_REGISTER_ONE_HALF =
+        PS_REGISTER_ZERO |
+        PS_INPUTMAPPING_HALFBIAS_NEGATE, // invalid for final combiner
+    PS_REGISTER_NEGATIVE_ONE_HALF =
+        PS_REGISTER_ZERO |
+        PS_INPUTMAPPING_HALFBIAS_NORMAL, // invalid for final combiner
 };
 
-enum PS_COMBINERCOUNTFLAGS
-{
-    PS_COMBINERCOUNT_MUX_LSB=     0x0000L, // mux on r0.a lsb
-    PS_COMBINERCOUNT_MUX_MSB=     0x0001L, // mux on r0.a msb
+enum PS_COMBINERCOUNTFLAGS {
+    PS_COMBINERCOUNT_MUX_LSB = 0x0000L, // mux on r0.a lsb
+    PS_COMBINERCOUNT_MUX_MSB = 0x0001L, // mux on r0.a msb
 
-    PS_COMBINERCOUNT_SAME_C0=     0x0000L, // c0 same in each stage
-    PS_COMBINERCOUNT_UNIQUE_C0=   0x0010L, // c0 unique in each stage
+    PS_COMBINERCOUNT_SAME_C0 = 0x0000L, // c0 same in each stage
+    PS_COMBINERCOUNT_UNIQUE_C0 = 0x0010L, // c0 unique in each stage
 
-    PS_COMBINERCOUNT_SAME_C1=     0x0000L, // c1 same in each stage
-    PS_COMBINERCOUNT_UNIQUE_C1=   0x0100L  // c1 unique in each stage
+    PS_COMBINERCOUNT_SAME_C1 = 0x0000L, // c1 same in each stage
+    PS_COMBINERCOUNT_UNIQUE_C1 = 0x0100L // c1 unique in each stage
```

Comment:
Please re-align as it suggests these belong together (I'd re-align each pair individually).